src/main/java/com/redhat/darcy/webdriver/internal/ForwardingTargetedWebDriver.java

```diff
@@ -21,6 +21,7 @@
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchSessionException;
```

Comment:
I think we don't need to add this import since you (smartly) found this is caught underneath NotFoundException